executor/batch_point_get.go

```diff
@@ -313,14 +313,9 @@ func (e *BatchPointGetExec) initialize(ctx context.Context) error {
 	rc := e.ctx.GetSessionVars().IsPessimisticReadConsistency()
 	// Lock keys (include exists and non-exists keys) before fetch all values for Repeatable Read Isolation.
 	if e.lock && !rc {
-		lockKeys := make([]kv.Key, len(keys), len(keys)+len(indexKeys))
+		lockKeys := make([]kv.Key, len(keys)+len(indexKeys))
```

Comment:
For the RC isolation level, seems the unique index keys will not be locked?

Reply:
RC isolation should lock exist keys now.